README.md

```diff
@@ -17,38 +28,55 @@ For the best integration into the Symfony full stack framework, it is
 recommended to use the [RoutingBundle](https://github.com/symfony-cmf/RoutingBundle)
 when building Symfony full stack applications.
 
-This library is provided by the [Symfony Content Management Framework (CMF) project](http://cmf.symfony.com/)
-and licensed under the [MIT License](LICENSE).
-
 
 ## Requirements
 
-* PHP 5.6 / 7
-* The Symfony Routing component (2.8 - 3.\*)
+* PHP 5.6 / 7.0 / 7.1
+* Symfony 2.8 / 3.1 / 3.2 / 3.3
 * See also the `require` section of [composer.json](composer.json)
 
-
 ## Documentation
 
 For the install guide and reference, see:
 
-* [Routing component documentation](http://symfony.com/doc/master/cmf/components/routing/index.html)
+* [symfony-cmf/routing Documentation](http://symfony.com/doc/master/cmf/components/routing/index.html)
 
 See also:
 
 * [All Symfony CMF documentation](http://symfony.com/doc/master/cmf/index.html) - complete Symfony CMF reference
 * [Symfony CMF Website](http://cmf.symfony.com/) - introduction, live demo, support and community links
 
+## Support
+
+For general support and questions, please use [StackOverflow](http://stackoverflow.com/questions/tagged/symfony-cmf).
 
 ## Contributing
 
 Pull requests are welcome. Please see our
 [CONTRIBUTING](https://github.com/symfony-cmf/symfony-cmf/blob/master/CONTRIBUTING.md)
 guide.
 
-Unit and/or functional tests exist for this component. See the
+Unit and/or functional tests exist for this bundle. See the
```

Comment:
this is not a bundle, but a component. lets change the template to say `package` - that is correct for anything, be it a bundle or not.